src/pages/gettingstarted/angular.hbs

```diff
@@ -0,0 +1,368 @@
+        <p>
+          We assume that your project is processing
+          <a href="https://sass-lang.com/" class="drizzle-b-Link">Sass</a> rather
+          than plain CSS for style information. More information
+          for converting an Angular project to SCSS is available at
+          <a href="https://angular.io/guide/ajs-quick-reference" class="drizzle-b-Link">angular.io</a>.
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          We assume that your Angular project is using Angular 6 with Typescript or is an Angular-CLI based
+          project. Currently,
+          <span class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              spark-core-angular
+            </code>
+          </span>
+          is dependent on Angular 6 and has not been fully tested using Angular 7. 
+        </p>
+      </li>
+    </ul>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2 sprk-u-mbm">Spark Core</h2>
+    <h4 class="drizzle-b-h4">Getting Started</h4>
+    <p>In order to get started with
+      <span class="drizzle-c-Command">
+        <code class="drizzle-CodeHighlight">
+          spark-core-angular
+        </code>
+      </span>, you'll need to follow these steps:
+    </p>
+
+    <ul class="drizzle-b-List drizzle-b-List--nested sprk-u-mbm">
+      <li class="sprk-u-mvm">
+        <p>Install the 
+          <span class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              spark-core
+            </code>
+          </span>  
+        npm package.</p>
+        <div class="drizzle-c-Command">
+          <code class="drizzle-CodeHighlight">
+            npm install --save-dev @sparkdesignsystem/spark-core</code>
+        </div>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>Install the
+          <span class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              spark-core-angular
+            </code>
+          </span>  
+        npm package.</p>
+        <div class="drizzle-c-Command">
+          <code class="drizzle-CodeHighlight">
+            npm install --save-dev @sparkdesignsystem/spark-core-angular</code>
+        </div>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Install the needed peer dependencies for our
+          npm package. When you use the Angular CLI it
+          should automatically install our peer
+          dependencies besides the packages listed below. If your application
+          does not have one of our peer dependencies
+          then your console should show a warning for that missing
+          dependency when you install 
+          <span class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              spark-core-angular
+            </code>
+          </span>
+          .
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Install additional peer dependencies:
```

Comment:
these are listed as dev deps in the spark-core-angular project, so they shouldnt have to manually install them

Reply:
Removed reference to additional dependencies